src/Request.php

```diff
@@ -141,7 +141,8 @@ public function request($access, $request, $endpointurl)
 
                         return $responseInstance->setText($body)->setResponse($xml);
                     } elseif ($xml->Response->ResponseStatusCode == 0) {
-                        throw new InvalidResponseException('Failure: '.$xml->Response->Error->ErrorDescription.' ('.$xml->Response->Error->ErrorCode.')');
+                        $code = $xml->Response->Error->ErrorCode + 0;
```

Comment:
What does the `+ 0` do here?

Reply:
You have to do that to convert it to a long integer.  You can't just pass the ErrorCode directly to Exception.

Reply:
What about using `intval` or `(int)`?